Features.md

```diff
@@ -0,0 +1,48 @@
+
+## Support status
+
+This is a (non-exhaustive) list of features currently (un)supported by CHIP-SPV.
+
+### Host side
+
+#### Unsupported / unimplemented APIs
+
+* hipRtc API
+* hipGraph API
+* hipIpc API
+* hipModuleOccupancy API
+* hipTexRef (texture reference) API
+* surface object / reference APIs
+* hipMemcpyPeer, hipMemcpyPeerAsync, hipMemRangeGetAttribute,
+  hipDeviceSetCacheConfig, hipDeviceGetCacheConfig,
+  hipDeviceSetSharedMemConfig, hipDeviceGetSharedMemConfig,
+  hipSetDeviceFlags,  hipGetDeviceFlags,
+  hipDeviceCanAccessPeer, hipDeviceEnablePeerAccess,
+  hipDeviceDisablePeerAccess, hipDeviceGetStreamPriorityRange,
+  hipStreamGetPriority, hipDevicePrimaryCtxRelease, hipDevicePrimaryCtxRetain,
+  hipDevicePrimaryCtxSetFlags, hipMemPrefetchAsync, hipMemAdvise,
+  hipModuleLoadData, hipModuleUnload, hipModuleLaunchKernel
+
+#### partially supported
+  * Texture Objects of 1D/2D type are supported; 3D, LOD, Grad,
+    Cubemap, Gather and Mipmapped textures are not supported
+
+### Device side
+
+#### Unsupported / unavailable
```

Comment:
What about all the failing compilation tests for devicelib?

Reply:
if you mean `Unit_deviceFunctions_CompileTest_*` from the HIP testsuite, those will be disabled in another commit in the next PR.

Reply:
> HIP testsuite 
you probably mean the HIP unit tests. The HIP-Testsuite is another repo entirely.

Reply:
yes sorry, HIP unit tests.